engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/formatter/ConfluentAvroMessageFormatter.scala

```diff
@@ -22,9 +23,18 @@ private[confluent] class ConfluentAvroMessageFormatter(schemaRegistryClient: Sch
 
   schemaRegistry = schemaRegistryClient
 
-  def writeTo(data: Array[Byte], output: PrintStream): Unit = {
-    val obj = deserialize(data)
+  def getSchemaIdOpt(bytes: Array[Byte]): Option[Int] = {
```

Comment:
Why this method is here? It looks strange when some class has multiple public methods that are not connected. WDYT about moving it to `ConfluentUtils.readIdOpt`? Or maybe just inline invocation: `Option(bytes).map(ConfluentUtils.readInt)`. Why this check: `avroContent.nonEmpty` is necessary? Why not to just throw exception like in this inlined example?

Reply:
Fixed.